src/app.mjs

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+const filePath = './src/test.txt';
+const folderPath = './';
+const enc = 'utf-8';
+
+async function copyFile(pathFrom, pathTo, encoding) {
+  const normalizedPathFrom = path.normalize(path.dirname(pathFrom));
+  const normalizedPathTo = path.normalize(pathTo);
+
+  if (normalizedPathFrom === normalizedPathTo) {
+    return;
+  }
+
+  // check is file exist
```

Comment:
Remove comment. Try it write obvious code that explains all we need without comments)
Fix all cases